app/controllers/pull_requests_controller.rb

```diff
@@ -15,17 +16,13 @@ def index
   def show
     github_user
     @pull_request = PullRequest.find(params[:id])
-    @reviewers_id = PullRequestReviewRequest.where(pull_request_id: @pull_request.id)
-    i = 0
-    @reviewers = []
-    loop do
-      @reviewers.push(GithubUser.where(id: @reviewers_id[i].github_user_id))
-      if (i + 1) == @reviewers_id.length
-        break
-      end
-
-      i += 1
-    end
+    @serialized_pull_request = PullRequestSerializer.new(@pull_request)
+    @likes_given = Like.where(github_user_id: github_user.id, likeable_type: "PullRequest")
+    @liked = @likes_given.where(likeable_id: @pull_request.id)[0]
```

Comment:
En el show no estás usando el `likes_given`, sería mejor hacer esto:

```suggestion
    @liked = Like.find_by(github_user_id: github_user.id, likeable_type: "PullRequest", likeable_id: @pull_request.id)
```

Reply:
buenísimo!! eso voy a hacer, es verdad que es mucho mejor 😄

Reply:
Al final no lo pude dejar como boleado porque necesito el id del like si es que lo quiero borrar, pero estaba buena la idea y lo simplifiqué ;)